feature/home/src/main/java/io/filmtime/feature/home/HomeScreen.kt

```diff
@@ -101,6 +153,80 @@ fun VideoSectionRow(
   }
 }
 
+@Composable
+fun UnknownErrorScreen() {
+  Column(modifier = Modifier.fillMaxSize(),
+    horizontalAlignment = Alignment.CenterHorizontally,
+    verticalArrangement = Arrangement.Center){
+
+    Text(text = "Unknown error occurred",
+      modifier = Modifier.padding(start = 20.dp))
+
+
+    }
+  }
+
+@Composable
+fun ApiErrorScreen(onRetry:()->Unit) {
+
+  Column(modifier = Modifier.fillMaxSize(),
+    horizontalAlignment = Alignment.CenterHorizontally,
+    verticalArrangement = Arrangement.Center){
+
+    Image(painter = painterResource(id = R.drawable.apierror),
+      contentDescription ="not_found",
+      modifier = Modifier.clip(shape = CircleShape)
+        .size(150.dp)
+
+    )
+    Spacer(modifier = Modifier.height(8.dp))
+    Text(text = "Oops! Something went wrong. Please try again later.",
+      modifier = Modifier.padding(start = 20.dp))
+    Button(onClick = onRetry,modifier = Modifier.clip(RoundedCornerShape(4.dp))) {
+      Text(text = "Retry")
+
+    }
+
+  }
+}
+
+@Composable
+fun NetworkErroScreen(context: Context,onRetry:()->Unit) {
```

Comment:
Why we need to pass context to this composable?

why not defining context on this composable near its usage?